app/perfsymboltable.cpp

```diff
@@ -736,11 +749,16 @@ static PerfAddressCache::SymbolCache cacheSymbols(Dwfl_Module *module, quint64 e
         GElf_Word shndx;
         quint64 start = 0;
         quint64 size = 0;
+        quint64 adjust = 0;
+        bool relocated = false;
         const auto symbol = dwfl_module_getsym_info(module, i, &sym, &symAddr, &shndx, &elfp, nullptr);
         if (symbol) {
-            start = sym.st_value;            
+            start = sym.st_value;
+            start = (isArmArch && (start & 1)) ? start - 1 : start;
```

Comment:
this ARM specific code change seems to be unrelated to this patch and should probably go to a separate one, or the previous one that introduced the `start` value in the symbol